<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DtBom</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="partListSelect">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>All</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>No shopid</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Diff</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="partList">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QToolButton" name="webBack">
            <property name="text">
             <string>&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="webFprward">
            <property name="text">
             <string>&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="webShopSelect"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWebView" name="webView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>5</height>
       </size>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="format">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuLoad">
     <property name="title">
      <string>Import</string>
     </property>
     <addaction name="actionImport_from_DipTrace_ASCII"/>
     <addaction name="actionImport_from_CSV"/>
    </widget>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionExport_to_CSV"/>
     <addaction name="actionExport_to_CSV_grouped"/>
     <addaction name="actionExport_to_CSV_by_shop"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuLoad"/>
    <addaction name="menuExport"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Stat</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionParts_group_by_shop"/>
    <addaction name="actionFile_info"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuInfo"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionImport_from_DipTrace_ASCII">
   <property name="text">
    <string>Import from DipTrace ASCII</string>
   </property>
  </action>
  <action name="actionRefDes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RefDes</string>
   </property>
  </action>
  <action name="actionValue">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Value</string>
   </property>
  </action>
  <action name="actionType">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </action>
  <action name="actionImport_from_CSV">
   <property name="text">
    <string>Import from CSV</string>
   </property>
  </action>
  <action name="actionExport_to_CSV">
   <property name="text">
    <string>Export to CSV</string>
   </property>
  </action>
  <action name="actionParts_group_by_shop">
   <property name="text">
    <string>Parts group by shop</string>
   </property>
  </action>
  <action name="actionFile_info">
   <property name="text">
    <string>File info</string>
   </property>
  </action>
  <action name="actionExport_to_CSV_grouped">
   <property name="text">
    <string>Export to CSV grouped</string>
   </property>
  </action>
  <action name="actionExport_to_CSV_by_shop">
   <property name="text">
    <string>Export to CSV by shop</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>webBack</sender>
   <signal>clicked()</signal>
   <receiver>webView</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>webFprward</sender>
   <signal>clicked()</signal>
   <receiver>webView</receiver>
   <slot>forward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
